### ISSUE TITLE
error running role on ubuntu

### ISSUE THREAD

**Issue by narayannarvekar** [andrewrothstein/ansible-etcd#1]
Andrew

I downloaded the etcd role and was trying to run it no Ubuntu (14.04 LTS) and had the following issues. I tried creating the required folder and got rid of the first error. Also tried to uncomment the missing variable name from default - but it started giving a different error.

Let me know how I can resolve this. I am using ansible 2.0.1.0

Thanks
Narayan

TASK [andrewrothstein.etcd : install etcd.service systemd unit] ****************
failed: [52.91.46.115] => (item=usr/lib/systemd/system/etcd.service) => {"changed": true, "failed": true, "item": "usr/lib/systemd/system/etcd.service", "msg": "Destination directory /usr/lib/systemd/system does not exist"}
failed: [52.91.46.115] => (item=etc/etcd/etcd.conf) => {"failed": true, "item": "etc/etcd/etcd.conf", "msg": "AnsibleUndefinedVariable: 'etcd_network_iface' is undefined"}

error after uncommenting the missing variable  'etcd_network_iface'

TASK [andrewrothstein.etcd : install etcd.service systemd unit] ****************
ok: [52.91.46.115] => (item=usr/lib/systemd/system/etcd.service)
failed: [52.91.46.115] => (item=etc/etcd/etcd.conf) => {"failed": true, "item": "etc/etcd/etcd.conf", "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute u'ansible_enp0s8'"}

**Comment by andrewrothstein** [comment]
First, apologies for not setting the meta data correctly. Over the past month I've enhanced the role to configure etcd clusters and not just install the etcd/etcdctl executables. I'm targeting CentOS 7 in production. I decided that for all processes that I'm going to be running on the infrastructure outside of Docker, I want to target systemd for process supervision. Ubuntu Trusty still uses upstart. Ubuntu Xenial uses systemd I believe.

https://github.com/andrewrothstein/ansible-etcd/commit/290ce1b1395df8329217c7b2752f57045992fddb

If it's Ubuntu Trusty or bust for you, I'm happy to accept a pull request that configures etcd tp be supervised by upstart.
